### PR TITLE
window: new tab client

### PR DESCRIPTION
**Pull request by mPaella**
## Description

new tab window client

## Test plan

local, tested setting pay button to open in new tab

### CODE REVIEW

File: packages/client/base/src/services/hosted/v3/crossmintHostedCheckoutV3Service.ts [hunk 1]
```diff
@@ -27,37 +27,37 @@ export function crossmintHostedCheckoutV3Service({
...
     }
 
-    // TODO: Implement new tab client
-    function createNewTabClient(): ReturnType<typeof PopupWindow.initSync> {
-        throw new Error("Not implemented");
+    function createNewTabClient(url: string): ReturnType<typeof NewTabWindow.initSync> {
+        return NewTabWindow.initSync(url, {});
     }
 
     // TODO: Implement same tab client
```
Comment (pgarciaegido) [inline packages/client/base/src/services/hosted/v3/crossmintHostedCheckoutV3Service.ts:42]: can we remove todo?

File: packages/client/window/src/windows/NewTab.ts [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        const popup = createNewTabSync(url);
+        return new NewTabWindow<IncomingEvents, OutgoingEvents>(
+            popup,
+            options.targetOrigin || urlToOrigin(url),
+            options
+        );
+    }
+}
+
+function createNewTabSync(url: string) {
```
Comment (pgarciaegido) [inline packages/client/window/src/windows/NewTab.ts:48]: could this be private methods instead of just functions?

**Review by pgarciaegido — APPROVED** [review]

File: packages/client/base/src/services/hosted/v3/crossmintHostedCheckoutV3Service.ts (hunk 1 above)
Comment (mPaella) [reply packages/client/base/src/services/hosted/v3/crossmintHostedCheckoutV3Service.ts:42]: thats on the next if, which isnt implemented yet

File: packages/client/window/src/windows/NewTab.ts (hunk 2 above)
Comment (mPaella) [reply packages/client/window/src/windows/NewTab.ts:48]: ehh i dont think it really makes a difference, and this is the pattern for all of them...

**Pull request merged**